python/python/glide/async_commands/cluster_commands.py

```diff
@@ -257,3 +258,21 @@ async def client_getname(
             TClusterResponse[Optional[str]],
             await self._execute_command(RequestType.ClientGetName, [], route),
         )
+
+    async def dbsize(self, route: Optional[Route] = None) -> int:
+        """
+        Returns the number of keys in the database.
+        See https://redis.io/commands/dbsize for more details.
+
+        Args:
+            route (Optional[Route]): The command will be routed to all primaries, unless `route` is provided,
+            in which case the client will route the command to the nodes defined by `route`.
+
+        Returns:
+            int: The number of keys in the currently selected database.
+
+        Examples:
+            >>> await client.dbsize()
+                10  # Indicates there are 10 keys in the current database.
```

Comment:
not accurate.  When routing isn't provided we're sending this command to multiple nodes (all primaries), the comment should say that